src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -378,4 +378,8 @@ private AccountResponse generateAccountResponse(Boolean fakeActive, Date fakeDat
         accountResponse.accountHolders = fakeAccountHolders;
         return accountResponse;
     }
+
+    private <T> List<T> makeListMutable(List<? extends T> inputList) {
+        return new ArrayList<>(inputList);
+    }
 }
```

Comment:
This fixed the test failing and the implementation still works.

Reply:
Makes sense per our Teams discussion. My bad 😄